src/services/apes/donations/donations.ts

```diff
@@ -27,12 +27,12 @@ export const donations_api = apes.injectEndpoints({
       DonationsQueryParams
     >({
       providesTags: ["donations"],
-      query: ({ id, chain_id, type, ...rest }) => {
+      query: ({ id, chain_id, status, ...rest }) => {
         return {
           url:
-            type === "finalized"
-              ? `${v(3)}/donation/${chain_id}/${id}`
-              : `v1/donations/on-hold/${id}`,
+            status === "PENDING"
+              ? `v1/donations/on-hold/${id}`
+              : `${v(3)}/donation/${chain_id}/${id}`,
```

Comment:
The check as it is in this PR allows any component that calls this endpoint to *not* provide the status if they want to load *finalized* donations, which are assumed to be desired by default.
I wasn't certain which approach to take: 
1. Make `status` optional and make the endpoint load finalized donations by default (if `status` not set).
2. Make `status` optional and make the endpoint load pending donations by default (if `status` not set).
3. Make `status` mandatory, thus requiring any component that targets the endpoint specify which donations they need (finalized vs pending).

I chose 1) and what you're suggesting is 2) and it would require more or less the same number of line changes (just in other places that load finalized donations, as we'll need to set the status there). Wanted to clarify that, but I'm open to suggestions.

Reply:
@misicnenad Thanks for clarifying that! Option 1 _does_ seem like the better approach. We'd more often want finalized donations (considering donors & nonprofit views both need that). Feel free to revert to original along with "PENDING".